apps.yaml

```diff
@@ -340,3 +340,18 @@ nmrium:
     categories:
         - data-visualization
         - data-analysis
+dft-qe:
+    metadata:
+        title: DFT-QE
+        description: |
+            The DFT-QE WaNo implements various methods available within the Quantum Espresso code.
+        authors: Celso R. C. Rêgo
+        external_url: https://simstack.readthedocs.io/en/latest/
+        documentation_url: https://github.com/KIT-Workflows/DFT-QE/edit/main/README.md
```

Comment:
This link should go to the correct readme and not the editable version, consider replacing the `edit` in the url with `blob`.